eclipse/site/pom.xml

```diff
@@ -171,7 +171,7 @@
       <plugin>
         <groupId>org.jboss.tools.tycho-plugins</groupId>
         <artifactId>repository-utils</artifactId>
-        <version>${jbossTychoPluginsVersion}</version>
+        <version>0.23.1</version>
```

Comment:
tbh I have no clue. We can try to switch to 0.26.0 later but that is out of scope of this PR. The whole Maven Reactor of the SY tools is messed up and should be cleaned up soon.